tests/unit/lms/services/blackboard_api/client_test.py

```diff
@@ -73,12 +70,13 @@ def test_it_with_pagination(self, svc, basic_client, blackboard_list_files_schem
             ),
             factories.requests.Response(json_data={}),
         ]
+        blackboard_files = [MagicMock() for _ in range(8)]
```

Comment:
```suggestion
        blackboard_files = [{"id": id_} for id_ in range(8)]
```

If we want things which allow dict access, why not use a dict? Bare mocks don't really show the reader what the code is working with.

Reply:
Probably better to use dicts (haven't done it yet). It will need to contain all the values needed for the event.